terraform-infra-approvals/global.json

```diff
@@ -5,7 +5,8 @@
     {"type": "azurerm_application_insights"},
     {"type": "azurerm_storage_account"},
     {"type": "azurerm_storage_container"},
-    {"type": "random_string"}
+    {"type": "random_string"},
+    {"type": "azurerm_template_deployment"}
```

Comment:
this defeats the purpose of the approval really, it means you can run anything in the ARM template, this should either be wrapped in a module or reworked to use the terraform resource

Reply:
Yeah, this would leave a massive hole where teams could do anything. It would be best to wrap in a module or use terraform resource as Tim says. If this can't be achieved in 2 weeks, then, in the short term, this could be added to a repo-specific whitelist file rather than global.

Reply:
Agreed as discussed adding it temporarily to repo specific json file. Will create a tech debt ticket which will extract the code into a module.